website_sale_cart_preview/static/src/js/website_sale_cart_preview.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
```

Comment:
There is no doc to support this way of doing this feature, nevertheless its the most efficent way to load this kind of JS files is after browser finishes retrieveing all base JS  and not abuse over the ```.ready();``` method, this does not mean that you are wrong or I am right, just is not a good practice, its more about a shape rule, anyway thanks for the great contribution, lets merge it.